salt-master/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN wget -O - https://repo.saltstack.com/apt/ubuntu/ubuntu14/latest/SALTSTACK-GP
     echo "deb http://repo.saltstack.com/apt/ubuntu/ubuntu14/latest trusty main" >> /etc/apt/sources.list && \
     apt-get update && apt-get install -y salt-master && apt-get install -y salt-minion && \
     /bin/echo -e "tcp_keepalive_idle: 300\ntcp_keepalive: True\n" >> /etc/salt/minion && \
+    mkdir -p /srv/salt/_libs/base /srv/salt/salt-formulas /srv/pillar
 
 ADD docker/salt/ /srv/salt/
```

Comment:
Using COPY rather than ADD would create the dest for you and is also the "better" command to use